### ISSUE TITLE
SaveEvent. notifySuccess(); Unable to refresh the tab page of the channel

### ISSUE THREAD

**Issue by liuxxin** [MicrosoftDocs/msteams-docs#9970]
[在此处输入反馈]
Calling the saveEvent. notifySuccess() method using pages. config. registerOnSaveHandler on the tab page of the channel did not trigger the tab refresh of the channel page. The same usage method can trigger tab page refresh on personal tab pages

---
#### 文档详细信息

⚠ *请勿编辑此部分。 learn.microsoft.com ➟ GitHub 问题链接必须具有此部分。*

* ID: 54495010-ec9f-281a-8d5e-15faa2e2b847
* Version Independent ID: c2b3283e-aef2-d84e-4201-7b392ad79cdc
* Content: [config module - Teams](https://learn.microsoft.com/zh-cn/javascript/api/@microsoft/teams-js/pages.config?view=msteams-client-js-latest)
* Content Source: [nodejs/docs-ref-autogen/@microsoft/teams-js/pages.config.yml](https://github.com/OfficeDev/msteams-client-docs-ref-typescript/blob/live/nodejs/docs-ref-autogen/@microsoft/teams-js/pages.config.yml)
* Service: **msteams**
* GitHub Login: @charlesprakash
* Microsoft Alias: **o365devx**

**Comment by sayali-MSFT** [comment]
@liuxxin - Thanks for reporting your issue.
We will check this at our end and will get back to you.

**Comment by liuxxin** [comment]
> @liuxxin- 感谢您报告您的问题。 我们将在最后检查这一点，然后回复您。

Okay, thank you

**Comment by sayali-MSFT** [comment]
Hello @liuxxin -Could you please confirm are you facing same issue as mentioned here?
https://github.com/OfficeDev/microsoft-teams-library-js/issues/2034

**Comment by liuxxin** [comment]
> 你好@liuxxin- 您能否确认您是否面临与此处提到的相同问题？ [OfficeDev/microsoft-teams-library-js#2034](https://github.com/OfficeDev/microsoft-teams-library-js/issues/2034)

The problem we are facing is that we open the tab settings function, pop up a pop-up, click the save button on the pop-up, click save on the personal tab page, close the pop-up, and refresh the current tab on the personal page. Similarly, we open the settings on the channel and click the save button. At this time, the pop-up will close directly but the current tab will not be refreshed. It seems that our situation is not quite the same

**Comment by sayali-MSFT** [comment]
@liuxxin -Could you please confirm, are you trying the same thing like below, as the channel tab is not refreshed after click on save button -

https://github.com/MicrosoftDocs/msteams-docs/assets/93527375/4c4111b9-3ec1-4fbd-8715-047e09ab6660

**Comment by liuxxin** [comment]
> @liuxxin-您能否确认一下，您是否正在尝试像下面一样的操作，因为单击“保存”按钮后频道选项卡不会刷新 -
> 
>  录音.2023-11-23.171727.mp4

Yes, I tried the above operation, but the same operation can refresh the tab in the personal tab.
![Uploading 20231124-090436.gif…]()

**Comment by liuxxin** [comment]
> @liuxxin-您能否确认一下，您是否正在尝试像下面一样的操作，因为单击“保存”按钮后频道选项卡不会刷新 -
> 
>  录音.2023-11-23.171727.mp4

Yes, I tried the above operation, but the same operation can refresh the tab in the personal tab.

https://github.com/MicrosoftDocs/msteams-docs/assets/66517994/6c6386d2-e34c-4beb-979d-97bd1c137b41

**Comment by liuxxin** [comment]
> @liuxxin -Could you please confirm, are you trying the same thing like below, as the channel tab is not refreshed after click on save button -
> 
>  Recording.2023-11-23.171727.mp4

Yes, I tried the above operation, but the same operation can refresh the tab in the personal tab.

**Comment by sayali-MSFT** [comment]
@liuxxin -We are able to repro that issue, we have raised bug for the same. We will inform you once we get any update.

**Comment by liuxxin** [comment]
> @liuxxin-我们能够重现这个问题，我们已经提出了同样的错误。一旦收到任何更新，我们会通知您。

OK, thanks
